apps/src/dance/p5.dance.js

```diff
@@ -461,8 +461,12 @@ exports.getEnergy = function getEnergy(range) {
   }
 }
 
+function getCurrentTime() {
```

Comment:
We may want to export this, to make it available for https://github.com/code-dot-org/code-dot-org/pull/25358#issuecomment-429085239.